lib/helpers/statusMap.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
```

Comment:
As this file is only use (by now) in the ohos adapter, placing it in a different file may cause the webpack to bundle this unused code. Im not sure, can you test it?

Reply:
thanks, I will test it later

Reply:
Now that I saw it, should the statusText format be `IN_THAT_WAY`?

Reply:
Did you get this tested?